dandiapi/api/views/webdav/__init__.py

```diff
@@ -80,8 +80,23 @@ def atpath(request: Request):
     if dandiset.embargoed and not is_dandiset_owner(dandiset, request.user):
         raise PermissionDenied
 
-    qs = get_atpath_queryset(version=version, path=params['path'], children=params['children'])
+    # This endpoint requires an ending slash for folders,
+    # but the atpath implementation requires the opposite
```

Comment:
Was this comment copied from somewhere else?  This *is* the atpath endpoint, and it doesn't require (or prohibit) an ending slash.

Reply:
Good catch, that was a typo that I've now fixed. I meant to refer to the `AssetPath` implementation, which is what this endpoint utilizes.